client/src/components/addName.js

```diff
@@ -25,12 +28,20 @@ function AddName() {
     // allow only letters, periods, spaces and hyphens
     event.target.value =
       event.target.value.replace(/[^a-zA-Z. -]+/gi, '') || '';
+    setInputValue(event.target.value);
     clearErrors('name');
   };
 
+  //Capitalize the first letter of the user's input:
+  const capitalize = (input) => {
+    const value = input.trim();
+    return value.charAt(0).toUpperCase() + value.slice(1);
+  };
+
   const onSubmit = (formData) => {
     // POST API call
-    fetch(`${process.env.REACT_APP_HOST}/names`, {
+    fetch(`http://localhost:3001/names`, {
+      // fetch(`${process.env.REACT_APP_HOST}/names`, {
```

Comment:
Switch this back before merging.

Reply:
I will switch back.  I was using my local docker container to test these changes.